.github/workflows/reusable-test.yaml

```diff
@@ -103,8 +103,10 @@ jobs:
           scope: "@agntcy"
 
       - name: Test SDKs
+        working-directory: ./server
         run: |
           task sdk:deps:javascript
+          go mod download
           task server:start &
           sleep 60 # Wait for dir-apiserver startup
           task sdk:test:all
```

Comment:
lets not rebuild everything again to start the server. you can use the built image to launch the server or a task command that spawns the server via helm chart. check how this was solved for e2e tests